### PR TITLE
The holy pull-request :bomb:

### PR DESCRIPTION
**Pull request by beydogan**
@xaph So are you going review this?

deal

### CODE REVIEW

File: app/controllers/admin/admin_controller.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  layout "admin_layout"
+  rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
+
+  before_filter :set_current_conference
+
+  attr_accessor :current_conference
+
+  def set_current_conference
+    subdomain = request.subdomain
+    black_list = ["", "www"]
```
Comment (xaph) [inline app/controllers/admin/admin_controller.rb]: we can add smtp, mail, imap, ftp black list keywords to here. but not now :)

File: app/views/admin/conferences/dashboard/_activity_item.html.erb [hunk 2]
```diff
@@ -13,7 +13,7 @@
                 <p class="nm">
                   <%= t("activities.actions.#{activity.action}",
                         user: "<span class=semibold>#{activity.user.display_name}</span>",
-                        speech: link_to(activity.subject.topic.subject, [activity.conference, activity.subject.commentable])
+                        speech: link_to(activity.subject.topic.subject, ["admin", activity.subject.commentable])
```
Comment (xaph) [inline app/views/admin/conferences/dashboard/_activity_item.html.erb]: we may use :admin instead of :admin

Comment (beydogan) [inline app/views/admin/conferences/dashboard/_activity_item.html.erb]: @xaph 
:admin instead of :admin ? 

hayırdır

Comment (xaph) [inline app/views/admin/conferences/dashboard/_activity_item.html.erb]: I wanted to say "admin" :) :+1:

**Pull request merged**